newIDE/app/src/UI/ClosableTabs.js

```diff
@@ -19,7 +19,7 @@ const styles = {
     width: '100%',
     position: 'relative',
     textAlign: 'initial',
-    height: '100%',
+    minHeight: 0,
```

Comment:
Let's verify in all editors (scene editor, events editor, debugger, extension, external layout, external events) that they still display properly :)

Reply:
Looks okay to me!